### PR TITLE
Website trial

### PR DESCRIPTION
**Pull request by tadawson23**
- HTML file in the folder 'template' which allows user input
- API that currently reads POST requests to terminal
- Has been adjusted such that it accepts percentages rather than prices

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+from flask import Flask, render_template, request
+
+app = Flask(__name__, template_folder='templates')
+
+submissions = []
+
+
+@app.route('/')
+def index():
+    return render_template('input_website.html')
```
Comment (Abbey-Wilkinson) [inline app.py]: Could use a docstring?

File: app.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+
+@app.route('/submit', methods=["POST"])
+def submit():
+    if request.method == 'POST':
```
Comment (Abbey-Wilkinson) [inline app.py]: Again here could use a docstring describing what the function does

File: app.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            'first_name': first_name,
+            'last_name': last_name,
+            'email': email,
+            'url': url,
+            'discount': discount
+        }
+
+        submissions.append(data)
+
+        print("Form Submitted:")
```
Comment (HarvindG) [inline app.py:34]: does this print to the browser or the console?

Comment (tadawson23) [reply app.py:34]: Prints to the console for the time being

**Review by Abbey-Wilkinson — APPROVED** [review]
Brilliant 👍

**Pull request merged**